### PR TITLE
Support rotation counter setting in EncoderMotor class

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/<ticket-number>) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: packages/pma/pitop/pma/encoder_motor.py [hunk 1]
```diff
@@ -286,15 +284,35 @@ def rotation_counter(self):
...
         )
         output_shaft_rotation_counter = round(
-            dc_motor_rotation_counter / self.MMK_STANDARD_GEAR_RATIO, 1
+            adjusted_dc_motor_rotation_counter / self.MMK_STANDARD_GEAR_RATIO, 1
         )
 
         return output_shaft_rotation_counter
 
+    @rotation_counter.setter
+    def rotation_counter(self, rotations):
```
Comment (angusjfw) [inline packages/pma/pitop/pma/encoder_motor.py:298]: i think for our learning users it will be easier to understand a method motor.resetRotationCount() than a setter, which looks like a variable to them.  resetRotationCount could have basically the same implementation with a defaut of 0 and optional rotations number
